recipes/libyaml/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
```

Comment:
Unchanged from bincrafters

Reply:
it's correct :+1: Actually, 90% of recipes use it.